tailwind.config.ts

```
@@ -58,5 +60,5 @@ export default {
   		}
   	}
   },
-  plugins: [require("tailwindcss-animate")],
+  plugins: [require("tailwindcss-animate"),heroui()],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)